Nos/Views/Modifiers/View+StyledBorder.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import SwiftUI
+
+extension View {
+    func withStyledBorder() -> some View {
+        modifier(StyledBorder())
+    }
+}
+
+struct StyledBorder: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .overlay(
+                RoundedRectangle(cornerRadius: 16)
+                    .strokeBorder(
+                        LinearGradient(
+                            gradient: Gradient(colors: [
+                                Color("styled-border-gradient-top"),
+                                Color("styled-border-gradient-bottom")
+                            ]),
+                            startPoint: .top,
+                            endPoint: .bottom
+                        ),
+                        lineWidth: 3
+                    )
+            )
+            .shadow(
+                color: Color(white: 0, opacity: 0.15),
```

Comment:
Now that we have #1375 done, we'll probably want to create a new color constant for this. Maybe this would be named something like "styled-border-shadow" in the Colors.xcassets, then you should be able to use it here like this:

```suggestion
                color: .styledBorderShadow,
```